sellotape/sellotape_dj/main_app/admin.py

```diff
@@ -1,3 +1,27 @@
 from django.contrib import admin
+from django.contrib.auth.models import User
 
 # Register your models here.
+
+# https://docs.djangoproject.com/en/dev/topics/auth/customizing/#extending-the-existing-user-model
+class User(models.Model):
+	# Inherits Attributes from User default instance
+    user 	= models.OneToOneField(User, on_delete=models.CASCADE)
+    # Generates a user ID which is the primary identifier to the user
+    user_id	= models.AutoField(primary_key=True)
```

Comment:
As I've noted on the `user_followers_id` below, it's unnecessary to manually add the primary key.